kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
The solution was about a build container. It should not interfere with local development environment. Basically that is my suggestion for build-gen-contianer.sh for a specific language:

1. Read pom file of that language and extract swagger-codegen version from it.
2. If it is not a final release (has - in it), parse it and extract commit hash from it. (if it said SNAPSHOT then head would be the commit hash).
3. Build a container by checking out that specific commit (or a release version or head).
4. Copy and then modify language pom.xml into the container to replace the commit hash (if exists) with "-SNAPSHOT"
5. Setup rsync's and generate the client using this container.